.github/workflows/integration.yml

```diff
@@ -232,7 +232,7 @@ jobs:
         # Produces ghcr.io/agoric/agoric-sdk:unreleased used in the following upgrade test.
         # run: cd packages/deployment && ./scripts/test-docker-build.sh | $TEST_COLLECT
         # XXX skip TAP test output and collection for now; it hides the output from the logs
-        run: make docker-build-sdk
+        run: make GIT_COMMIT=unspecified GIT_REVISION=unspecified docker-build-sdk
```

Comment:
This was necessary to build it the same before and after the `restore-node`. @mhofman why is it important to check that?

`yarn build` in a3p now builds the SDK itself so I wonder if we can cut out these steps